djangoTrading/settings.py

```diff
@@ -68,16 +82,48 @@
     },
 ]
 
+REST_FRAMEWORK = {
+    'DEFAULT_FILTER_BACKENDS': (
+        'django_filters.rest_framework.DjangoFilterBackend',
+    ),
+    # 'DEFAULT_PERMISSION_CLASSES': (
+    #     'rest_framework.permissions.IsAuthenticated',
+    # )
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        'rest_framework_jwt.authentication.JSONWebTokenAuthentication',
+        'rest_framework.authentication.SessionAuthentication',
+        'rest_framework.authentication.BasicAuthentication',
+    ),
+}
+
+# REDIS related settings
+# REDIS_HOST = '0.0.0.0'
+# REDIS_PORT = '6379'
+REDIS_HOST = env('REDIS_HOST')
+REDIS_PORT = env('REDIS_PORT')
+# CELERY_ALWAYS_EAGER = False
+CELERY_BROKER_URL = 'redis://' + REDIS_HOST + ':' + REDIS_PORT + '/0'
```

Comment:
**DO NOT** use + operator on strings. Better use:
Format strings: 
```python
f'redis://{REDIS_HOST}:{REDIS_PORT}/0'
```
Format method:
```python
'redis://{}:{}/0'.format(REDIS_HOST, REDIS_PORT)
```